src/main/java/ch/securify/CompilationHelpers.java

```diff
@@ -58,7 +58,8 @@ public static String sanitizeLibraries(String hexCode) {
     static int bytecodeOffsetToSourceOffset(int bytecodeOffset, List<String[]> map) throws NotFound {
         try {
             map = map.subList(0, bytecodeOffset);
-        } catch (IndexOutOfBoundsException e) {
+        } catch (IndexOutOfBoundsException | IllegalArgumentException e) {
+            e.printStackTrace();
             throw new NotFound();
```

Comment:
This `NotFound` seems strange. Imported from some pretty unrelated package, why are we using that?
Printing the stacktrace is not necessarily good here, it seems to me like catching these exceptions is part of normal program flow... e.g. other parts catching the NotFound later and handling it accordingly.

Reply:
True, fixed that.